crates/providers/src/anilist/mod.rs

```diff
@@ -120,6 +122,20 @@ impl NonMediaAnilistService {
     }
 }
 
+impl Display for media_details_query::MediaStatus {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let str = match self {
+            media_details_query::MediaStatus::FINISHED => "Finished".to_string(),
+            media_details_query::MediaStatus::RELEASING => "Ongoing".to_string(),
+            media_details_query::MediaStatus::NOT_YET_RELEASED => "Not Yet Released".to_string(),
+            media_details_query::MediaStatus::CANCELLED => "Canceled".to_string(),
+            media_details_query::MediaStatus::HIATUS => "Hiatus".to_string(),
+            _ => "Unknown".to_string()
```

Comment:
Can we just set it to `None` if the status is not known.

Also if we do that, then I think implementing the `Display` trait will not be required. A function will suffice.